projects/components/src/data-exporter/data-exporter.component.html

```diff
@@ -44,10 +44,10 @@ <h3 class="modal-title">{{ dialogHeader | lazyString }}</h3>
         </ng-container>
         <ng-template #purifyDataButtons>
             <div class="modal-footer">
-                <button type="button" class="btn btn-outline no" (click)="_downloadData(_data, false)">
+                <button type="button" class="btn btn-outline no" (click)="downloadData(data, false)">
                     {{ noText | lazyString }}
                 </button>
-                <button type="button" class="btn btn-primary yes" (click)="_downloadData(_data, true)">
+                <button type="button" class="btn btn-primary yes" (click)="downloadData(data, true)">
```

Comment:
It looks like we don't really need to pass data here if we're already storing it as state? Then we could make `data` private?